### ISSUE TITLE
BatchSize parameter setting

### ISSUE THREAD

**Issue by sanakhamekhem** [0x454447415244/HandwritingRecognitionSystem#9]
I would like to set suitability the training parametrers for a dataset of 8000 line images:
BatchSize and NEpochs.

**Comment by 0x454447415244** [comment]
Hello, I did not understand your question. You could use a batch size of 10 or 20 images (depending on your GPU memory) with a learning rate of 0.0005 or 0.001 respectively.

**Comment by sanakhamekhem** [comment]
Hello, thank you for your reply. Really, I have used a bach size of 64. The training stage is progressing. Thanks.
